oca_dependencies.txt

```diff
@@ -0,0 +1 @@
+purchase_order_block https://github.com/Eficent/purchase-workflow 11.0-mig-purchase_order_block-refactor
```

Comment:
What PR does this depend on?

Reply:
@lasley - This one: https://github.com/OCA/purchase-workflow/pull/480

Reply:
Thanks @rgarnau. In the future, please note this in your PR description in order to highlight the dependencies. I went ahead and did it here.

Reply:
That's true! Forgot! Thanks @lasley!